src/Impostor.Api/Net/Messages/S2C/Message00HostGameS2C.cs

```diff
@@ -12,7 +12,7 @@ public static void Serialize(IMessageWriter writer, int gameCode)
             writer.EndMessage();
         }
 
-        public static GameOptionsData Deserialize(IMessageReader reader)
+        public static LegacyGameOptionsData Deserialize(IMessageReader reader)
```

Comment:
overzealous refactor tool?

Reply:
yes, but that signature is incorrect for that message in the first place